packages/editor/src/components/post-title/post-title-raw.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { TextareaControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { store as editorStore } from '../../store';
+import { DEFAULT_CLASSNAMES, REGEXP_NEWLINES } from './constants';
+
+const PostTitleRaw = () => {
```

Comment:
There is still some duplication here but I'm not sure it's yet worth the effort of extracting shared logic to a hook. LMK if you think I should @ellatrix?